source/common/config/bootstrap_json.cc

```diff
@@ -130,5 +131,20 @@ void BootstrapJson::translateBootstrap(const Json::Object& json_config,
   }
 }
 
+void BootstrapJson::translateStaticSecretsBootstrap(
```

Comment:
We won't support SDS in v1 config. Please remove all v1 config references in this PR.

Reply:
Removed the method and related test cases